stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -464,4 +464,4 @@ function print_native_tokens(io, tokens, arch::Union{Val{:x86}, Val{:arm}})
         end
         print(io, spaces)
     end
-end
+end
```

Comment:
Please restore the line ending
```suggestion
end

```